custom/icds_reports/models/views.py

```diff
@@ -711,3 +711,85 @@ class Meta(object):
         app_label = 'icds_model'
         managed = False
         db_table = 'awc_location_months'
+
+
+class DishaIndicatorView(models.Model):
+    awc_id = models.TextField(primary_key=True)
+    awc_name = models.TextField(blank=True, null=True)
+    supervisor_id = models.TextField(blank=True, null=True)
+    supervisor_name = models.TextField(blank=True, null=True)
+    block_id = models.TextField(blank=True, null=True)
+    block_name = models.TextField(blank=True, null=True)
+    district_id = models.TextField(blank=True, null=True)
+    district_name = models.TextField(blank=True, null=True)
+    state_id = models.TextField(blank=True, null=True)
+    state_name = models.TextField(blank=True, null=True)
+    aggregation_level = models.IntegerField(blank=True, null=True)
+    month = models.DateField(blank=True, null=True)
+    # agg_awc Indicator
+    cases_household = models.IntegerField(blank=True, null=True)
```

Comment:
If this is the only thing you need from `agg_awc`, you can add it to the `agg_awc_monthly` view to remove a join

Reply:
@calellowitz

Reply:
It looks like that is already in the `agg_awc_monthly` view

Reply:
Cool, I have updated.